pkg/payment/search_request.go

```diff
@@ -1,42 +1,40 @@
 package payment
 
-import (
-	"net/url"
-)
+import "strings"
 
 // SearchRequest is the request to search services.
 // Filters field can receive a lot of paramaters. For details, see:
 // https://www.mercadopago.com.br/developers/pt/reference/payments/_payments_search/get.
 type SearchRequest struct {
+	Filters map[string]string
+
 	Limit  string
 	Offset string
-
-	Filters map[string]string
 }
 
-// Parameters transforms SearchRequest into url params.
-func (s SearchRequest) Parameters() string {
-	params := url.Values{}
-
-	for k, v := range s.Filters {
-		params.Add(k, v)
+// Check sets values for limit and offset when not sent.
+func (s *SearchRequest) Check() {
```

Comment:
`Check` traz o significado de apenas fazer uma verificação e não definir nada
```suggestion
// SetDefaults sets values for limit and offset when not sent.
func (s *SearchRequest) SetDefaults() {
```

O mesmo vale para as demais ocorrências de `Check`

Reply:
feito!